Clarify disabling MFA message

kevinlinxc opened the pull request:
I found the "Disable it first" MFA message to be a bit confusing, especially right after signing in with an OTP. I tried clarifying it (English + French)
<img width="544" alt="image (6)" src="https://user-images.githubusercontent.com/44324811/172667180-0717d493-f5f6-46bd-9579-f83dc9515405.png">

bettymakes reviewed (commented):
🇬🇧 English copy looks 👍 . Only thing that crossed my mind is whether RubyGems prefers `you'll` vs. `you will`. The current tone with `you'll` is more conversational. TBH, I prefer that. However, if RubyGems favours a formal tone, then `you will` would make more sense.

(PS I quickly skimmed through this file to get a sense of conversational vs. formal and it seems to vary. In short, I think either option is fine. ✅)

---

🇫🇷 French copy - I'm not a native speaker so my vote doesn't count. I was in the "Extended French Program" throughout middle school and high school 😅 . Based on that knowledge, it looks good to me. But I'll defer to a native French speaker 🙈. 

Tagging @maximecb for French copy review 👀 🙏

jenshenny commented:
Thank you for the update on the message, it clarifies the intent a lot better 🙌  It looks like there are translations for `es`, `zh-CN` and `zh-TW` for the old message, not sure what to do about those.

My guess is to remove them and wait to re-translate them.

deivid-rodriguez commented:
The Spanish translation could be something like

> Se ha activado la autenticación de múltiples factores. Para reconfigurarla, primero tendrás que desactivarla.

I went with the same approach as the French copy, not mentioning MFA explicitly, but using a pronoun to reference it instead.

simi commented on config/locales/en.yml:
```diff
@@ -292,7 +292,7 @@ en:
     incorrect_otp: Your OTP code is incorrect.
     otp_code: OTP code
-    require_mfa_disabled: Your multifactor authentication has been enabled. You have to disable it first.
+    require_mfa_disabled: Your multifactor authentication has been enabled. To reconfigure MFA, you'll have to disable it first.
```
:information_source: We should not use MFA - https://github.com/rubygems/rubygems.org/pull/2685 + we should use `multi-factor` instead of `multifactor` (per https://github.com/rubygems/rubygems.org/pull/3093/commits/0b598ce695aa79665bef8be8cb367bc50b570e9c).

kevinlinxc commented:
- MFA-> spelled out
- Added hyphens to English and french (https://www.linguee.com/english-french/translation/multi-factor+authentication.html has hyphens in multi-facteur)
- Added spanish, thank you
- Removed zhs

kevinlinxc commented:
Rebased to fix conflict

kevinlinxc commented:
Rebased and squashed
